dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -307,8 +309,234 @@ public void findOneTestWrongUUID() throws Exception {
+
+        //2. One public item
+        Item item = ItemBuilder.createItem(context, col1)
+                               .withTitle("Public item 1")
+                               .withIssueDate("2017-10-17")
+                               .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                               .withSubject("ExtraEntry")
+                               .withWritePermission(true)
+                               .build();
+
+        // A token must be provided for withdraw operation. The person
+        // is used in the provenance note.
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        // withdraw item
+        getClient(token).perform(patch("/api/core/items/" + item.getID())
+            .header(AUTHORIZATION_HEADER, AUTHORIZATION_TYPE + " " + token)
+            .content("[{\"op\":\"replace\",\"path\":\"/withdrawn\", \"value\": true}]")
+            .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$.uuid", Matchers.is(item.getID().toString())))
+                        .andExpect(jsonPath("$.withdrawn", Matchers.is(true)))
+                        .andExpect(jsonPath("$.inArchive", Matchers.is(false)));
+
+        // item already withdrawn, no-op, 200 response
+        getClient(token).perform(patch("/api/core/items/" + item.getID())
+            .content("[{\"op\":\"replace\",\"path\":\"/withdrawn\", \"value\": true}]")
+            .contentType(MediaType.APPLICATION_JSON_PATCH_JSON))
+                        .andExpect(status().isOk());
+
+    }
+
+    @Test
+    public void valueMissingForWithdrawalOperation() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1)
+                                           .withRemovePermission(true)
+                                           .withName("Collection 1").build();
+
+        //2. One public item
+        Item item = ItemBuilder.createItem(context, col1)
+                               .withTitle("Public item 1")
+                               .withIssueDate("2017-10-17")
+                               .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                               .withSubject("ExtraEntry")
+                               .withWritePermission(true)
```

Comment:
instead than introduce support for write/remove permission please use an administrator account to run the test (see also comments below)